### PR TITLE
Feature/converter

### PR DESCRIPTION
**Pull request by RenataY**

### CODE REVIEW

File: src/App.jsx [hunk 1]
```diff
@@ -91,6 +102,21 @@ const App = () => {
           <Route path="/about" element={<LandingMain />}></Route>
           <Route path="/contact" element={<LandingMain />}></Route>
           <Route path="/" element={<LandingMain />}></Route>
+          <Route path="/currencyconverter"
```
Comment (shushpan-v-satori) [inline src/App.jsx:105]: the Transfer component shall be refactored in order to support that functionality

File: src/App.jsx [hunk 2]
```diff
@@ -14,15 +14,26 @@ import BillingAddress from "./components/BillingAddress/BillingAddress";
...
 import ContactListPage from "./components/ContactListPage/ContactListPage";
 import LiveRates from "./components/LiveRates/LiveRates";
+import CurrencyConverterContainer from "./containers/CurrencyConverterContainer/CurrencyConverterContainer";
+import currency from './data/currency';
+
 
 const App = () => {
   const [searchTerm, setSearchTerm] = useState("");
-
+  const [baseCurrency, setBaseCurrency] = useState(currency[0]);
```
Comment (shushpan-v-satori) [inline src/App.jsx:23]: Hardcoded values will need to be refactored afterwards to allow use to choose another currency

File: src/components/CurrencyConverter/CurrencyConverter.jsx [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+    const [shouldHide, setShouldHide] = useState(true);
+    const [inverseFXRate,setInverseFXRate] = useState(Number(1/fxRate).toFixed(4))
+    
+    const currencyFromName = baseCurrency.name;
+    const currencyToName = toCurrency.name;
+    const messageFrom = `${amount} ${currencyFromName} =`;
+    const messageTo =  `${convertedAmount} ${currencyToName}`;
+    const message1From = `1 ${baseCurrency.code} = ${1 * fxRate} ${toCurrency.code}`;
+    const message1To = `1 ${toCurrency.code} = ${inverseFXRate} ${baseCurrency.code}`;
+    const numericRegExp = /^[0-9]+$/;
```
Comment (shushpan-v-satori) [inline src/components/CurrencyConverter/CurrencyConverter.jsx:21]: numbers are not accepting decimal value, need to change the regex

File: src/App.jsx (hunk 2 above)
Comment (monicagtaa) [reply src/App.jsx:23]: -Consider a drop down on the to currency.
-The base currency is determine by the Live Rate with 1
- Functionality for the converter needs to be revised. For example, If bank account (i.e., my base currency ) is in USD, and I need to send EUR 75.14, I need to be able to switch the currencies, but beep my base currency intact (i.e., FROM currency)

**Comment by monicagtaa** [comment]
see above comments

**Review by monicagtaa — APPROVED** [review]

**Pull request merged**